### ISSUE TITLE
Crash when pasting twice in some JS editors in UIWebView.

### ISSUE THREAD

**Issue by mteep** [SafeExamBrowser/seb-mac#288]
Certain JavaScript editors (Quill) in the "classical" web view (`UIWebView`) may cause SEB to crash when an attempt is made to paste text twice using keyboard shortcuts on iPad.

The first paste action has been observed to make the editor lose focus. As a result, the following paste action has nowhere to go, which causes a `paste:` message to be sent to the `UIWebView`. Its (default) implementation of `doesNotRecognizeSelector:` throws an exception which isn't caught, causing the app to crash.

I have a fix ready where I simply override `doesNotRecognizeSelector:` in the existing `UIWebView` category to do nothing, except log.

**Comment by mteep** [comment]
This issue was observed in AAC mode on iPadOS 15.7.3. (The fix hasn't been tested in single app mode since I don't have the entitlement.)

**Comment by danschlet** [comment]
Anther bug in UIWebView... Overriding `doesNotRecognizeSelector:` in the category sounds trivial, I'll definitely test it for the next release. Thanks for the report!

**Comment by mteep** [comment]
Yes, it is trivial. I added a pull request for this also. But feel free to adjust and re-arrange as you please.

**Comment by mteep** [comment]
I should add that the crash as stated was encountered while I tried to reproduce another crash in the wild on SEB 3.2.2 as configured by an Inspera exam.

I only have the original crash report, not the SEB configuration or log (since this was on a managed device prohibited from using Apple Mail accounts).

Finally I was able to reproduce something close enough by opening [Quill playground](https://quilljs.com/playground/) in SEB 3.2.4 using UIWebView and pasting twice in the editor.

Of course, the editor shouldn't lose focus after pasting. But focus issues seem common with JavaScript in UIWebView. Occasionally focus needs to be regained by touch. In order to do that, the app must not crash.

After I had created a fix, I found that this seems related to problems observed a long time ago, as can be seen [here](https://stackoverflow.com/questions/21316784/uiwebview-bug-uiwebview-cut-unrecognized-selector-sent-to-instance#21320320). But the suggested fixes there seem less safe than simply ignoring the unrecognized selectors.
